app/forms/support_interface/application_forms/edit_gcse_award_year_form.rb

```diff
@@ -1,13 +1,14 @@
 module SupportInterface
   module ApplicationForms
-    class EditGcseForm
+    class EditGcseAwardYearForm
       include ActiveModel::Model
 
       attr_reader :gcse
       attr_accessor :award_year, :audit_comment
 
-      validates :award_year, presence: true
+      validates :award_year, presence: true, length: { is: 4 }
```

Comment:
Looks like we have an existing `YearValidator` we could use here? Think this will also check that the year isn't in the future?

```suggestion
      validates :award_year, year: true, future: true
```